### ISSUE TITLE
Publicly exposed baseURL

### ISSUE THREAD

**Issue by kvnkim** [FoxxMD/multi-scrobbler#150]
- multi-scrobbler: [e.g. 0.7.0]

**Additional context**
Hello, thanks for this project it's working great as expected, however I was a tad concerned about the security of the web server being exposed (not on the default port of 9078)

I don't have the ability to configure a reverse proxy or http auth due to shared host restrictions on the vps (though I assume that would break webhooks altogether and the purpose of the whole thing), is it possible to have it not launch the web server and log only? i'm not too concerned atm, the only thing a person could do is restart the brainz client I have configured.  Any suggestions?

**Comment by FoxxMD** [comment]
If you are running MS with the docker image then preventing access is as easy as not publishing the port when running the container (don't use `-p` in run command or comment out `ports` section in `docker-compose.yml`)

If you are running locally (with npm/node from the command line) I've added the ability to disable the web server in the `develop` branch/image:

* ENV `DISABLE_WEB=true`
* In All-in-One config (`config.json`) use the top-level property `"disableWeb": true`

If you do this correctly you'll see this in the log on startup

```
[2024-04-04 09:28:42.811 -0400] WARN   : [App] [API] API and Dashboard have been DISABLED. Note that any ingress sources (Plex, Jellyfin, Tautulli, etc...) will be unusable
```

**Comment by kvnkim** [comment]
Yeah, just as I thought it would break the webhooks/Tautulli as a source if the webserver was offline heh

**Comment by FoxxMD** [comment]
This feature is now available in [v0.7.1](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.7.1).
